release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1559,6 +1559,36 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
+
+* Previously, if it encountered a conflict while updating a resource quota, the API server sometimes failed to create a resource and returned a 409 status code. In that case, the client had to retry the API request. With this update, when the API server encounters this type of conflict, it retries creating the resource up to three times. This is often sufficient to create the resource. Otherwise, the API server displays an error in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1920699[BZ#1920699])
+
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run.
+This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[BZ#1970470])
+
+* Previously, when users inserted a formatted YAML snippet in the YAML editor, the new selection did not match the new content in the snippet: the indentation was removed, and some random letters remained behind the selection. The current release fixes this issue. Now, the cursor remains in the position where it started and adds the missing indentation for the cursor end position. After inserting the YAML snippet, the new selection matches the new content. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952545[BZ#1952545])
+
+* Previously, annotations were passed to the specification of knative services along with metadata. As a result, decorators were shown for associated revisions of knative services in the topology. The current release fixes this issue by passing annotations only to knative service metadata. Now, decorators are shown only for knative services in the topology and not for associated revisions.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1954959[BZ#1954959])
+
+* Previously, when selecting the *YAML* tab, the `metadata.managedFields` section was not collapsing immediately. This was due to an issue with the Form or YAML switcher for pages such as *Pipeline Builder* and *Edit HorizontalPodAutoscaler* (HPA). As a result, the part of the document where the user tried to type collapsed. The `metadata.managedFields` section remained as is, and the cursor was reset to the starting position to the top left of the YAML editor. The current release fixes this issue. Now, upon loading the YAML, the `metadata.managedFields` section is immediately collapsed.
```

Comment:
was not collapsing/did not collapse
Form and YAML should be in bold
section collapses immediately.

Reply:
Done